lib/resolvers/membership_request_mutations/accept_membership_request.js

```diff
@@ -12,7 +12,9 @@ module.exports = async (parent, args, context) => {
   });
   if (!membershipRequest) {
     throw new NotFoundError(
-      requestContext.translate('membershipRequest.notFound'),
+      process.env.NODE_ENV !== 'production'
```

Comment:
once your other branch has been merged in, can you also add this as well to the constants file?